inc/blocks/namespace.php

```diff
@@ -180,9 +183,35 @@ function register_post_type() {
 			'menu_icon' => 'dashicons-networking',
 			'menu_position' => 152,
 			'show_in_rest' => true,
+			'show_in_menu' => 'index.php',
 			'rest_base' => 'xbs',
 			'rest_controller_class' => __NAMESPACE__ . '\\REST_API\Posts_Controller',
 			'hierarchical' => true,
+			'capabilities' => [
+				'create_posts' => 'do_not_allow',
+			],
+			'labels' => [
+				'name' => __( 'Experience Insights', 'altis-analytics' ),
+				'all_items' => __( 'Insights', 'altis-analytics' ),
+			],
+			'admin_cols' => [
+				'block' => [
+					'title' => __( 'Block', 'altis-analytics' ),
+					'function' => '\\Altis\\Analytics\\Dashboard\\render_block_column',
+				],
+				'views' => [
+					'title' => __( 'Views', 'altis-analytics' ),
+					'function' => '\\Altis\\Analytics\\Dashboard\\render_views',
```

Comment:
Extended CPTs supports setting the default sort column here so you don't need to do any special handling yourself:
```suggestion
					'function' => '\\Altis\\Analytics\\Dashboard\\render_views',
					'default' => 'DESC',
```
https://github.com/johnbillion/extended-cpts/wiki/Admin-columns#default-sort-column